src/net.h

```diff
@@ -360,16 +360,16 @@ class CNode
 public:
     /** This node's max acceptable number ancestor transactions.  Used to decide whether this node will accept a
      * particular transaction. */
-    size_t nLimitAncestorCount = BCH_DEFAULT_ANCESTOR_LIMIT;
```

Comment:
as per the c++11 spec, setting a var like this in the header is equivalent to adding a constructor for this member in the implementation of every constructor that does not explicitly construct it.  That is, class C { int a = B; }  === class C { int a; C():a(B) {} }.

Doing it in the header is better though, because it ensures that the variable won't be forgotten to be initialized in any new constructor that's added.

I'd like to see the same done with the other new variables as well.

Reply:
ok done,  but I don't like the idea of not being consistent. We should probably tidy up net.cpp sometime soon.